app/screens/ScheduleScreen/ScheduleCard.tsx

```diff
@@ -252,10 +252,10 @@ const ScheduleCard: FC<ScheduleCardProps> = (props) => {
     isSecondaryCallout,
   } = props
   const navigation = useAppNavigation()
-  const onPress = !heading
-    ? undefined
-    : ["talk", "trivia-show"].includes(variant)
-    ? () => navigation.navigate("TalkDetails", { scheduleId: id })
+  const onPress = ["talk", "trivia-show"].includes(variant)
```

Comment:
@mazenchami and I discussed this over Zoom. We'd like to make some updates to the data layer that would make these types of conditionals unnecessary.